docs/source/tutorial.rst

```diff
@@ -350,14 +350,8 @@ in MLflow saved the model as an artifact within the run.
           curl -X POST "http://127.0.0.1:8090/predict/" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"fixed acidity\": 6.2, \"volatile acidity\": 0.66, \"citric acid\": 0.48, \"residual sugar\": 1.2, \"chlorides\": 0.029, \"free sulfur dioxide\": 29, \"total sulfur dioxide\": 75, \"density\": 0.98, \"pH\": 3.33, \"sulphates\": 0.39, \"alcohol\": 12.8}"
 
       which should return something like::
-
-        {
-          "predictions": [
-            [
-              6.1312
-            ]
-          ]
-        }
+      
+            [6.1312]
```

Comment:
Hi @bayethiernodiop, I just reran this example and got output of the form 

```
{"predictions":[[6.3524]]}
```

What MLflow version are you using? And just to confirm, did you get an output of `[6.1312]` while running the R or Python example?